lib/GridLayout.ts

```diff
@@ -746,6 +709,76 @@ export default class GridLayout extends HTMLElement {
     this.syncWithNode(layout);
   }
 
+  #collectLayoutState(): GridLayoutState["layout"] {
+    const layout: GridLayoutState["layout"] = [];
+    [...this.children].forEach((node) => {
```

Comment:
Why copy children?

Reply:
and we need slot which contains only widgets nodes

Reply:
> Why copy children?

`children` doesn't have method `forEach`. Replaced with `for ... of`



> and we need slot which contains only widgets nodes

Added check for `instance of GridLayoutElement`